### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by LuisSalas94**
**Hello Microverse Team**

_In this branch We:_
- Made sure that there are no linter errors
- Made sure that you used the correct Gitflow
- Made sure that you documented your work in a professional way
- Made sure that the N+1 problem is solved when fetching all posts and their comments for a user
- Used Capybara to write integration tests for each view in your project.

_Happy Codding_

### CODE REVIEW

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @LuisSalas94, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Tests are passing :+1: 

## Required Changes ♻️
- [ ] I would recommend including the steps to test the application in the README file. [Hint: `rspec`]

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I would recommend removing or commenting on the request test as these tests are failing which is evident because the application has been updated.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/views/posts/index.html.erb_spec.rb [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
+
+    it 'see number of posts user has written' do
+      visit user_path(@first_user)
+      expect(page).to have_content('0')
+    end
+
+    it 'see post title' do
+      visit user_path(@first_user)
+      expect(page).to have_content('Hello')
+    end
```
Comment (alexander16108) [inline spec/views/posts/index.html.erb_spec.rb:32]: for these test cases there are no test cases for when :
`Tests that assert you can see a section for pagination if there are more posts than fit on the view.`

`Tests that assert you can see some of the post's body.`

please write and conduct these test cases again 💯

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @LuisSalas94 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hello Team👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**